DetectedObject[] detectedObjects
int32 count
